server/routes/api/conversations.js

```diff
@@ -20,9 +20,9 @@ router.get("/", async (req, res, next) => {
         },
       },
       attributes: ["id"],
-      order: [[Message, "createdAt", "DESC"]],
+      order: [[Message, "createdAt", "ASC"]],
       include: [
-        { model: Message, order: ["createdAt", "DESC"] },
+        { model: Message, order: ["createdAt", "ASC"] },
```

Comment:
While this fixes the messages ordering, unfortunately, it results in the conversations becoming out of order! we want the conversations to stay ordered by the ones with the latest message on top